lnrpc/routerrpc/router_backend.go

```diff
@@ -231,13 +231,41 @@ func (r *RouterBackend) QueryRoutes(ctx context.Context,
 		return nil, err
 	}
 
+	// Calculate route success probability. Do not rely on a probability
```

Comment:
The PR description would be useful as the commit message also :)

Reply:
Added